ci/scripts/connect-release-init.js

```diff
@@ -0,0 +1,163 @@
+            const packageJSON = JSON.parse(rawPackageJSON);
+            const { version } = packageJSON;
+
+            const gitLogResult = exec('git', [
+                'log',
+                '--oneline',
+                '--max-count',
+                '10',
+                '--pretty=tformat:"-   %s (%h)"',
+                '--',
+                `./packages/${packageName}`,
+            ]);
+
+            const commitsArr = gitLogResult.stdout.split('\n');
+
+            const CHANGELOG_PATH = path.join(PACKAGE_PATH, 'CHANGELOG.md');
+
+            const newCommits = [];
+            for (let commit of commitsArr) {
+                if (commit.includes(`npm-release: @trezor/${packageName}`)) {
+                    break;
+                }
+                newCommits.push(commit.replaceAll('"', ''));
+            }
+
+            if (newCommits.length) {
+                if (!fs.existsSync(CHANGELOG_PATH)) {
+                    fs.writeFileSync(CHANGELOG_PATH, '');
+                }
+
+                let changelog = fs.readFileSync(CHANGELOG_PATH, 'utf-8');
+
+                changelog = `# ${version}\n\n${newCommits.join('')}\n\n${changelog}`;
+                fs.writeFileSync(CHANGELOG_PATH, changelog, 'utf-8');
+
+                exec('yarn', ['prettier', '--write', CHANGELOG_PATH]);
+            }
+
+            commit({
+                path: PACKAGE_PATH,
+                message: `npm-release: @trezor/${packageName} ${version}`,
+            });
+        }
+    }
+
+    exec('yarn', ['workspace', '@trezor/connect', 'version:patch']);
+
+    const PACKAGE_PATH = path.join(ROOT, 'packages', 'connect');
+    const PACKAGE_JSON_PATH = path.join(PACKAGE_PATH, 'package.json');
+    const rawPackageJSON = fs.readFileSync(PACKAGE_JSON_PATH);
+    const packageJSON = JSON.parse(rawPackageJSON);
+    const { version } = packageJSON;
+
+    const commitMessage = `npm-release: @trezor/connect ${version}`;
+    const branchName = `npm-release/connect-${version}`;
+
+    exec('git', ['checkout', '-b', branchName]);
+
+    commit({
+        path: ROOT,
+        message: commitMessage,
+    });
+
+    exec('git', ['push', 'origin', branchName]);
+
+    const ghPrCreateResult = exec('gh', [
+        'pr',
+        'create',
+        '--repo',
+        'trezor/trezor-suite',
+        '--title',
+        `${commitMessage}`,
+        '--body-file',
+        'docs/releases/connect-release.md',
+        '--base',
+        'develop',
+        '--head',
+        branchName,
+    ]);
+
+    const prNumber = ghPrCreateResult.stdout
+        .replaceAll('\n', '')
+        .replace('https://github.com/trezor/trezor-suite/pull/', '');
+
+    if (errors.length) {
+        comment({
+            prNumber,
+            body: `Deps error. one of the dependencies likely needs to be published for the first time: ${errors.join(
+                ', ',
+            )}`,
+        });
+    }
+
+    const depsChecklist = update.reduce(
```

Comment:
This looks very nice.
For clarification the idea of those checkboxes is so then we can manually check that those packages were updated properly and check the checkbox. Right?

Reply:
yes. you also see actual npm version in the PR and you can compare it with version in commit message - they should match